### PR TITLE
Change Agents config files (added cisco switch and ports info)

### PR DESCRIPTION
**Pull request by syedasifraza**

### CODE REVIEW

File: T1_US_FNAL/Agent01/main.yaml [hunk 1]
```diff
@@ -4,12 +4,12 @@ general:
   sitename: [T1_US_FNAL]
   webdomain: "https://cmssense1.fnal.gov:8443"
 agent: 
-  interfaces: [enp129s0f2]
+  interfaces: [enp129s0f1]
   hostname: cmssense2.fnal.gov
-enp129s0f2:
+enp129s0f1:
   port: "1_2"
```
Comment (juztas) [inline T1_US_FNAL/Agent01/main.yaml:10]: This needs to point to correct port

File: T1_US_FNAL/Agent02/main.yaml [hunk 2]
```diff
@@ -4,12 +4,12 @@ general:
   sitename: [T1_US_FNAL]
   webdomain: "https://cmssense1.fnal.gov:8443"
 agent: 
-  interfaces: [enp129s0f2]
+  interfaces: [enp129s0f1]
   hostname: cmssense3.fnal.gov
-enp129s0f2:
+enp129s0f1:
   port: "1_2"
```
Comment (juztas) [inline T1_US_FNAL/Agent02/main.yaml:10]: this also needs to point to correct port on the switch

**Review by juztas — CHANGES_REQUESTED** [review]
We need to change port to point to switch port name

**Review by syedasifraza — COMMENTED** [review]
Changes done

File: T1_US_FNAL/Agent01/main.yaml [hunk 3]
```diff
@@ -4,12 +4,12 @@ general:
   sitename: [T1_US_FNAL]
   webdomain: "https://cmssense1.fnal.gov:8443"
 agent: 
-  interfaces: [enp129s0f2]
+  interfaces: [enp129s0f1]
   hostname: cmssense2.fnal.gov
-enp129s0f2:
-  port: "1_2"
+enp129s0f1:
+  port: "1_25"
```
Comment (juztas) [inline T1_US_FNAL/Agent01/main.yaml:10]: Change it to: `Ethernet1/25`

File: T1_US_FNAL/Agent02/main.yaml [hunk 4]
```diff
@@ -4,12 +4,12 @@ general:
   sitename: [T1_US_FNAL]
   webdomain: "https://cmssense1.fnal.gov:8443"
 agent: 
-  interfaces: [enp129s0f2]
+  interfaces: [enp129s0f1]
   hostname: cmssense3.fnal.gov
-enp129s0f2:
-  port: "1_2"
+enp129s0f1:
+  port: "1_26"
```
Comment (juztas) [inline T1_US_FNAL/Agent02/main.yaml:10]: Change it to `Ethernet1/26`

**Review by juztas — CHANGES_REQUESTED** [review]

**Comment by juztas** [comment]
Looks good

**Pull request merged**